os/watch/src/main.scala

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
What is this files. Shouldn't it reside in the test module?

Reply:
Ops! Will remove it.

Reply:
https://github.com/com-lihaoyi/os-lib/pull/70/commits/c2c1c63aba37ef61159a1037bf31222488049458